components/Decks.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect } from "react";
+import { useState } from "react";
+import { View, Text, FlatList, StyleSheet } from "react-native";
+
+const Decks = () => {
+  const testDecks = [
+    { name: "JS", description: "Testtesttest", id: 1 },
+    { name: "Python", description: "Testtesttest", id: 2 },
+    { name: "HTML", description: "Testtesttest", id: 3 },
+  ];
+  const [currentDecks, setCurrentDecks] = useState([]);
+
+  useEffect(() => {}, []);
+
+  // const renderItem = ({ item }) => {
+  //   return (
+  //     <View style={deckStyles.deckList}>
+  //       <Text style={deckStyles.name}>{item.name}</Text>
+  //       <Text style={deckStyles.description}>{item.description}</Text>
+  //     </View>
+  //   );
+  // };
+
+  return (
+    <View style={deckStyles.container}>
+      {/* <FlatList
+        data={testDecks}
+        renderItem={renderItem}
+        keyExtractor={(item) => item.id}
+      /> */}
+      {testDecks.map((testDeck) => (
+        <View style={deckStyles.deckList} key={testDeck.id}>
+          <Text style={deckStyles.name}>{testDeck.name}</Text>
+          <Text style={deckStyles.description}>{testDeck.description}</Text>
+        </View>
+      ))}
+    </View>
+  );
+};
+
+const deckStyles = StyleSheet.create({
+  container: {
+    // height: "100%",
+    // width: "100%",
+    flex: 1,
+    flexDirection: "column",
+    backgroundColor: "#4682B4",
+    justifyContent: "center",
+    alignItems: 'stretch',
+    alignContent: 'stretch'
+  },
+  name: {
+    fontWeight: "bold",
+    fontSize: 32,
+    padding: 5,
+    margin: 10,
+    color: "#9381FF",
+  },
+  description: {
+    fontSize: 24,
+    padding: 5,
+    margin: 10,
+    color: "#9381FF",
+  },
+  deckList: {
+    borderStyle: "solid",
+    borderWidth: 5,
+    borderTopColor: "#4682B4",
+    borderLeftColor: "#4682B4",
+    borderRightColor: "#77486E",
+    borderBottomColor: "#77486E",
+    backgroundColor: "#F5F3E5",
+    padding: 10,
+    margin: 20,
+    height: 170,
+  },
+});
+
+export default Decks;
```

Comment:
Great Job overall. Found these 2 resources for playing inside the screen safe space -> ios = https://reactnative.dev/docs/safeareaview Android = https://stackoverflow.com/questions/51289587/how-to-use-safeareaview-for-android-notch-devices. In the case of our component be going out of screen :)

Reply:
Ah ok that isn't something I really know about so I'll have a look, cheers!